src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,4 +1,93 @@
 package com.kodilla.ecommercee.domain;
 
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Entity
+@Table(name = "PRODUCTS")
 public class Product {
+    private long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Group group;
+
+    public Product() {
+    }
+
+    public Product(String name, String description, BigDecimal price) {
+        this.name = name;
+        this.description = description;
+        this.price = price;
+    }
+
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    public long getId() {
+        return id;
+    }
+
+    @NotNull
+    @Column(name = "NAME")
+    public String getName() {
+        return name;
+    }
+
+    @Column(name = "DESCRIPTION")
+    public String getDescription() {
+        return description;
+    }
+
+    @NotNull
+    @Column(name = "PRICE")
+    public BigDecimal getPrice() {
```

Comment:
uzywajac BigDecimal, warto ustawic w bazie danych parametr scale i precision
https://stackoverflow.com/questions/35435691/bigdecimal-precision-and-scale